packages/block-editor/src/components/background-image-control/style.scss

```
@@ -83,6 +84,32 @@
 	height: 100%;
 	width: 100%;
 	padding-left: $grid-unit-15;
+	padding-right: $grid-unit-40;
+}
+
+.block-editor-global-styles-background-panel__reset {
+	position: absolute;
+	right: 0;
+	top: $grid-unit;
+	margin: auto $grid-unit auto;
+	opacity: 0;
+	transition: opacity 0.1s ease-in-out;
+	@include reduce-motion("transition");
```

```suggestion
	@media not ( prefers-reduced-motion ) {
		transition: opacity 0.1s ease-in-out;
	}
```
Nit: we don't use the mixin anymore. See: https://github.com/WordPress/gutenberg/issues/68282#issuecomment-2561848046

---

Thanks for the heads up!